omero/developers/Model/EveryObject.txt

```diff
@@ -11,7 +11,6 @@ reference document lists every OMERO model object and their more useful
 properties, with an emphasis on enumerating every direct relationship
 among the objects.
 
-
```

Comment:
As long as there's one blank line between text and headings, it just ignores additional ones